handbook/handbook.company.values.md

```diff
@@ -88,12 +88,38 @@ Iteration means that we will hit the deadline but are flexible about the scope.
 
 We look for virtuous cycles and long term growth in all that we do. This means we think about both the long term **benefits and costs** of the actions we take and invest in areas that will take many years to bear fruit. It also means we don't do things as a one off and try to find ways to re-use existing work. 
 
+### Key Principles
 
-### In Practice
-- We pay down technical debt by dedicating cycles to it every sprint
+#### Systems First
+
+While we generally trust that people will try to do the right thing, humans make mistakes. Its part of being human. Therefore, we aim to build automated processes and systems for common and complicated issues. 
+
+In Practice:
+- Continues Integration tests on all features
+- Checklists and SOPs for all processes
+
+#### Flywheels
+
+We are looking to create systems that make make things easier
+
+In Practice:
 - [[Lookup|handbook.company.values#lookup]] -> [[Iterate|handbook.company.values#iteration]] -> [[Learn|handbook.company.values#lifelong-learning]]
 
-## Passing It Forward
+#### Bias for Simplicity
+
+Things should be as complicated as they need to be and no more. 
+
+In Practice:
+- When creating new features, opt for sensible defaults vs need for extra configuration
+
+#### Pruning
```

Comment:
Really like how this one turned out. Will need to start living this key principle in it's entirety but it's so distinctly Dendron :) .